low-level/sensors/impl/generic/cpu_fault_sensor.py

```diff
@@ -101,9 +105,14 @@ def initialize(self, conf_reader, msgQlist, product):
             # Get the instance of the utility using ToolFactory
             self._utility_instance = self._utility_instance or \
                                 self.tool_factory.get_instance(cpu_fault_utility)
-        except Exception as e:
-            logger.error(f"Error while initializing, shutting down CPUFaultSensor : {e}")
-            self.shutdown()
+        except Exception as err:
+            err_msg = ("Error while initializing. "
+                f"Unable to get the instance of {cpu_fault_utility} Utility, {err}")
+            if self.recovery_enabled:
+                raise Exception(err_msg)
+            else:
+                logger.error(f"CPUFaultSensor: {err_msg}. Hence shutting down.")
```

Comment:
avoid using grammar like hence in log and all

Reply:
Done.